answers/database.md

```diff
@@ -70,12 +70,14 @@ Indexing adds a data structure with columns for the search conditions and a poin
 
 ## What algorithm and data structure indexing used? And why?
 
-Index can use a hash table or B-Tree.
+Index can use a hash table or B-Tree. [The index decided used depend on data type and how you query data](https://dev.mysql.com/doc/refman/8.0/en/index-btree-hash.html).
```

Comment:
@phamngocquy thank you for your contribution :pray: 
`The index decided used depend on data type and how you query data` It seems like the grammar of this sentence is not correct. Can you double check?

Reply:
@phamngocquy thank you!